Firebase/Auth/Source/FIRAuthErrorUtils.m

```diff
@@ -370,6 +382,18 @@
 static NSString *const kFIRAuthErrorMessageInvalidClientID = @"The OAuth client ID provided is "
     "either invalid or does not match the specified API key.";
 
+/** @var kFIRAuthErrorMessageWebRequestFailed
+    @brief Message for @c FIRAuthErrorCodeWebRequestFailed error code.
+ */
+static NSString *const kFIRAuthErrorMessageWebRequestFailed = @"A network error (such as timeout, "
+    "interrupted connection or ' + 'unreachable host) has occurred within the wb context.";
```

Comment:
The "' + '" part seems extraneous?

Also, I'm more in favor of Oxford comma to avoid ambiguity.